templates/scenarios/sso-tab-js/teamsfx/app.yml

```diff
@@ -31,7 +31,7 @@ deploy:
   - uses: azureStorage/deploy # Deploy bits to Azure Storage Static Website
     with:
       workingDirectory: .
```

Comment:
The working dir can be removed if working in the project path.

Reply:
thanks, resolved